src/main/java/com/jmb/springfactory/model/entity/Task.java

```diff
@@ -0,0 +1,56 @@
+package com.jmb.springfactory.model.entity;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import com.jmb.springfactory.model.enumeration.PriorityEnum;
+import com.jmb.springfactory.model.enumeration.TaskStatusEnum;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+public class Task extends BaseEntity {
+
+  private String name;
+
+  private String description;
+
+  private Double estimatedTime;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "estimatedTime" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)